policies/permissions.md

```diff
@@ -0,0 +1,100 @@
+matching their role (for example, "Astropy Core Maintainers" or "Astropy
+website maintainers" for a core maintainer or the astropy.org team, respectively),
+a duty primarily performed at the moment by the Coordination Committee.
+Temporary permissions should instead use the "collaborator" feature on Github to
+make it clear that these permissions are temporary in nature.
+
+Regardless of access level, even if it is temporary,
+[GitHub two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication)
+must be enabled for the affect user accounts.
+
+Additionally, the granter of permissions (usually the Coordination Committee)
+should send a message to the new recipient of write permissions listing the
+responsibilities and expectations that go with this - a template for this email
+is available [in this repo](../messages/maintainer_access.md). That message may
+contain a prompt for a response, which should be cc-ed/forwarded to
+coordinators@astropy.org .
+
+## Access levels
+
+### Core package maintainers
+
+All maintainers listed for the core package receive *write access* to the
+repository via the **Astropy Core Maintainers** GitHub team.
+
+### Coordinated package maintainers
+
+Coordinated package maintainers receive **admin access** to the coordinated
+package repositories via the **<package name> maintainers** GitHub team (e.g.,
+'astroquery maintainers').
+
+### Core package release coordinators
+
+Core package release coordinators receive **admin access** to the core
+repository, as well as the extension-helpers repository
+since releases of those packages may be tightly coupled, as
+well as **write access** to the website repository. This is done via the **Core
+release maintainers** GitHub team.
+
+### Coordination committee
+
+The Coordination Committee members receive **owner access** to
+the astropy GitHub organization. Members who are not familiar or
+comfortable with GitHub administration may opt out. However,
+a majority of the committee should have access. If necessary,
+members should receive GitHub administration training before given access.
+
+In addition, they have access to the project
+credentials (or the shared password manager to access the credentials).
+As with GitHub access above, members may opt out but the majority and training
+rules also apply here.
+
+Regardless of access level, the members are always bound by
+[APE 0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst).
+For example, a Coordination Committee member cannot delete or transfer
+a repository without first obtaining concensus from the community.
+
+## Other ways to gain access
+
+Besides the process laid out a the beginning of this document,
+which might not cover all cases, other ways include:
+
+### Automated access
+
+(TODO: Fill in info from the automated invite bot that Matt Craig deployed.)
```

Comment:
Waiting to hear back from @mwcraig

Reply:
When I have time to revisit this: https://github.com/astropy/astropy-tools/blob/main/.github/workflows/update_org_members.yml